device-types/Cisco/C9300X-12Y.yaml

```diff
@@ -0,0 +1,56 @@
+---
+manufacturer: Cisco
+model: Catalyst 9300X-12Y
+slug: cisco-c9300x-12y
+part_number: C9300X-12Y
+u_height: 1
+is_full_depth: true
+comments: '[Cisco Catalyst 9300 Series Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/catalyst-9300-series-switches/nb-06-cat9300-ser-data-sheet-cte-en.html)'
+console-ports:
+  - name: con 0
+    type: rj-45
+  - name: usb
+    type: usb-mini-b
+power-ports:
+  - name: PS-0
+    type: iec-60320-c14
+    maximum_draw: 715
+  - name: PS-1
+    type: iec-60320-c14
+    maximum_draw: 715
+interfaces:
+  - name: GigabitEthernet0/0
+    type: 1000base-t
+    mgmt_only: true
+  - name: TwentyFiveGigE1/0/1
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE1/0/2
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE1/0/3
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE1/0/4
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE1/0/5
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE1/0/6
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE1/0/7
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE1/0/8
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE1/0/9
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE1/0/10
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE1/0/11
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE1/0/12
+    type: 25gbase-x-sfp28
+  - name: HundredGigE1/0/13
+    type: 100gbase-x-qsfp28
+  - name: HundredGigE1/0/14
+    type: 100gbase-x-qsfp28
+  - name: HundredGigE1/0/15
+    type: 100gbase-x-qsfp28
+  - name: HundredGigE1/0/16
+    type: 100gbase-x-qsfp28
```

Comment:
This is a modular uplink, it should be modelled as such.  The name is also incorrect as it would be 1/1/#